### PR TITLE
fix bug in ibitset find_next method

### PR DESCRIPTION
**Pull request by Chiraffollo**

### CODE REVIEW

File: test/test_bitset.cpp [hunk 1]
```diff
@@ -1012,6 +1012,21 @@ namespace
       CHECK_EQUAL(4U, data.find_next(true,  1));
     }
 
+    //*************************************************************************
+    TEST(test_find_next_multi_byte_bitset)
+    {
+      etl::bitset<16> data;
+      data.set(12);
+      data.set(22);
```
Comment (jwellbelove) [inline test/test_bitset.cpp:1020]: 22 is larger that the size of the bitset. Did you mean to do that?

Comment (jwellbelove) [reply test/test_bitset.cpp:1020]: I'm nearly ready to push my fix, but I'll add your test too, when it's fixed.

Comment (Chiraffollo) [reply test/test_bitset.cpp:1020]: Sorry, that's my fault.
Simple rule: don't change anything between testing and commit.

Comment (Chiraffollo) [reply test/test_bitset.cpp:1020]: fixed

**Review by Chiraffollo — COMMENTED** [review]
test is fixed

**Pull request merged**